website/source/downloads_tools.html.erb

```diff
@@ -37,6 +37,14 @@ description: |-
   These Consul tools are created and managed by the amazing members of the Consul community:
 </p>
 <ul>
+ 
+  <li>
+    <a href="https://github.com/joeatbayes/file2consul">file2consul</a> - Mirror property files into consul.
+      go based config loader with support for multiple enviornments. Provides 
+      variable expansion, interpolation, inheritance with overrides and ability to update multiple consul servers.
+      Reduces cost of maintaining larger configuration sets between enviornments by reducing restatement and manual editing
+      of similar or predicably changing config properties.
+  </li>
```

Comment:
Hey, thanks for contributing this tool! We're really happy to add it to the list, do you mind putting it in the right spot alphabetically though to keep things orderly!